packages/strapi-plugin-upload/controllers/proxy.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
         headers: _.omit(ctx.request.headers, ['origin', 'host', 'authorization']),
       });
 
-      Object.entries(res.headers.raw()).forEach(([key, value]) => {
+      for (var [key, value] of res.headers.entries()) {
```

Comment:
Nice catch ! can you use const  instead of var :)

Reply:
Sure I can 👍